src/test/index.js

```diff
@@ -106,14 +106,13 @@ describe( 'Gutenberg Mobile initialization', () => {
 			{ locale: defaultLocale, capabilities },
 			{ component: EditorComponent }
 		);
-		const blockList = screen.getByTestId( 'block-list-wrapper' );
+		// Inner blocks create BlockLists so let's take into account selecting the main one
+		const blockList = screen.getAllByTestId( 'block-list-wrapper' )[0];
```

Comment:
These changes [were already introduced](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/src/test/index.test.js#L184-L198) in the Gutenberg repo.